### PR TITLE
feat assembly

### PR DESCRIPTION
**Pull request by xquanluu**

### CODE REVIEW

File: lib/tasks/transcribe.js [hunk 1]
```diff
@@ -293,6 +294,20 @@ class TaskTranscribe extends SttTask {
...
+      case 'assemblyai':
+        this.bugname = 'assemblyai_transcribe';
+        ep.addCustomEventListener(AssemblyAiTranscriptionEvents.Transcription,
+          this._onTranscription.bind(this, cs, ep, channel));
+        ep.addCustomEventListener(AssemblyAiTranscriptionEvents.Connect,
+          this._onVendorConnect.bind(this, cs, ep));
+        ep.addCustomEventListener(AssemblyAiTranscriptionEvents.ConnectFailure,
+          this._onVendorConnectFailure.bind(this, cs, ep, channel));
+
+        /* if app sets deepgramOptions.utteranceEndMs they essentially want continuous asr */
```
Comment (davehorton) [inline lib/tasks/transcribe.js:306]: why is this here

File: lib/utils/transcription-utils.js [hunk 2]
```diff
@@ -104,6 +105,9 @@ const stickyVars = {
   soniox: [
     'SONIOX_PROFANITY_FILTER',
     'SONIOX_MODEL'
+  ],
+  assemblyai: [
+    'ASSEMBLYAI_API_KEY'
```
Comment (davehorton) [inline lib/utils/transcription-utils.js:110]: word boost should cleared as well I think

**Review by davehorton — CHANGES_REQUESTED** [review]

File: lib/tasks/transcribe.js (hunk 1 above)
Comment (xquanluu) [reply lib/tasks/transcribe.js]: copy paste mistake, removed them

File: lib/utils/transcription-utils.js (hunk 2 above)
Comment (xquanluu) [reply lib/utils/transcription-utils.js]: added now

**Review by davehorton — APPROVED** [review]
